db/schema.rb

```diff
@@ -10,11 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_11_16_175220) do
+ActiveRecord::Schema.define(version: 2020_11_16_182332) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "bookings", force: :cascade do |t|
+    t.string "start_date"
+    t.string "days"
```

Comment:
As before, I think `days` should be an integer

Reply:
See above comment!